Tasks/JenkinsQueueJobV2/job.ts

```diff
@@ -404,9 +412,14 @@ export class Job {
         request.get({ url: fullUrl, strictSSL: thisJob.queue.TaskOptions.strictSSL }, function requestCallback(err, httpResponse, body) {
             tl.debug('streamConsole().requestCallback()');
             if (err) {
-                Util.handleConnectionResetError(err); // something went bad
-                thisJob.stopWork(thisJob.queue.TaskOptions.pollIntervalMillis, thisJob.State);
-                return;
+                if (thisJob.queue.TaskOptions.retryCount < 1) {
```

Comment:
Could we avoid duplication here?

Reply:
what exactly is duplicated? you mean join ifs?

Reply:
Yep, there are two very close if statements (except logging)- could we probably move it to separate method and pass the way we log it via callback?

![image](https://user-images.githubusercontent.com/8779682/130800789-32e301b0-dad1-432e-aab7-62e7f9c2e398.png)

Reply:
Not really similar, only thisJob.stopWork is similar wveryting else is different